primitives/src/merkle_tree/examples.rs

```diff
@@ -31,7 +31,7 @@ impl<F: RescueParameter> DigestAlgorithm<F, u64, F> for RescueHash<F> {
     }
 
     fn digest_leaf(pos: &u64, elem: &F) -> F {
-        let data = [F::from(*pos), *elem, F::zero()];
```

Comment:
This had to be changed to match the spec. 
Currently the MT-circuit is very specific for this MT data structure, we should either move it out of examples and re-export from crate root(?) as canonical MT or make it clear in the circuit implementation that it refers to append-only MT with 3-rate Rescue hasher.

Reply:
I just added a `prelude` module for mt_refactor. It now includes a canonical instantiation of `RescueMerkleTree`.

Reply:
Great, I'll rebase later.

Reply:
Done in https://github.com/EspressoSystems/jellyfish/pull/138/commits/b61fa4e8213813f0876159bc075764a02ece1440.